src/components/flat-table/flat-table-row/flat-table-row.spec.js

```diff
@@ -15,6 +15,9 @@ import StyledFlatTableCheckbox from "../flat-table-checkbox/flat-table-checkbox.
 import { SidebarContext } from "../../drawer";
 import FlatTableCheckbox from "../flat-table-checkbox";
 import StyledIcon from "../../icon/icon.style";
+import FlatTableRowHeader from "../flat-table-row-header/flat-table-row-header.component";
+import FlatTableHeader from "../flat-table-header/flat-table-header.component";
```

Comment:
I guess that we can import all components from one source as
```js
import { FlatTableRowHeader, FlatTableHeader, ...<more components> from '../flat-table' }
```

Reply:
I'm not sure it makes a huge difference in a spec file as it's not bundled etc

Reply:
oh right it is spec file :)